### PR TITLE
Support for log alert thresholds and config per platform

### PR DESCRIPTION
**Pull request by agopala95**
Introduced config files for AWS and GCP to have platform specific log alerts.

### CODE REVIEW

File: pkg/common/config/log_metrics.go [hunk 1]
```diff
@@ -23,14 +23,22 @@ var beforeSuiteMetrics = LogMetrics{}
 // GetLogMetrics will return the log metrics.
 func GetLogMetrics() LogMetrics {
 	once.Do(func() {
-		viper.UnmarshalKey("logMetrics", &logMetrics)
+		if CloudProvider.CloudProviderID == "gcp" {
```
Comment (jeefy) [inline pkg/common/config/log_metrics.go:26]: What if we removed the if statement and instead dynamically attempted to do UnmarshalKey? Like...

`err := viper.UnmarshalKey(fmt.Sprintf("%slogMetrics", CloudProvider.CloudProviderID)` and then if there's an error, log it and bubble it up?

**Review by jeefy — CHANGES_REQUESTED** [review]
Values look good and initial work looks good! One quick thought/change :)

File: pkg/common/config/log_metrics.go (hunk 1 above)
Comment (agopala95) [reply pkg/common/config/log_metrics.go]: Yup, this makes more sense. I've made the changes as suggested. Thanks!

**Comment by jeefy** [comment]
Thanks for the changes!
/approve
/assign @whereswaldon

**Pull request merged**